snippets/card-product.liquid

```diff
@@ -31,6 +33,8 @@
     if ratio == 0 or ratio == null
       assign ratio = 1
     endif
+    assign desktop = columns_desktop | at_least: 1
+    assign mobile = columns_mobile | at_least: 1
```

Comment:
Here we could use better descriptive naming I think. Those names don't convey clearly what the value is tied to. Maybe `columns_amount_desktop` 🤷 or something else